.github/actions/tests/api-tests/action.yaml

```diff
@@ -0,0 +1,46 @@
+name: 'Run API Tests'
+description: 'Sets up Python environment and executes API tests with pytest using specified markers'
+author: 'konrad'
+inputs:
+  user_api_secret:
```

Comment:
Does this need to be an input here? It is used in env up and even there it can be encapsulated within the reusable workflow.

Reply:
Tbh, I'm not sure how to do this with reusable actions. In reusable workflows I could pass the variables from one action to another. I do not think that there is an easy way on how to do this using reusable actions. Do you maybe have an example of how can I encapsulate it?

Reply:
I don't see the need for this to be an input at all. Afaik, this token needs to be the same between the data centres in the mdcb setup. It is handled in the dash-bootstrap script. Does this action fail if you remove this input here?

Reply:
user_api_secret is needed for mdcb api tests. We have tests that are interacting with org that uses mdcb setup